From: Carlos Garnacho <carlosg@gnome.org>
Date: Wed, 7 Apr 2021 12:02:17 +0200
Subject: libtracker-data: Change TrackerOntology timestamp to be gint64

So it is the same width for all platforms.

Origin: upstream, commit:74bf9d0a5d050988fb63172cc56583dcb64497a7
Applied-Upstream: 3.2.0~alpha
---
 src/libtracker-data/tracker-ontology.c | 6 +++---
 src/libtracker-data/tracker-ontology.h | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/libtracker-data/tracker-ontology.c b/src/libtracker-data/tracker-ontology.c
index edfc143..a050755 100644
--- a/src/libtracker-data/tracker-ontology.c
+++ b/src/libtracker-data/tracker-ontology.c
@@ -32,7 +32,7 @@ typedef struct _TrackerOntologyPrivate TrackerOntologyPrivate;
 
 struct _TrackerOntologyPrivate {
 	gchar *uri;
-	time_t last_modified;
+	gint64 last_modified;
 	gboolean is_new;
 	TrackerOntologies *ontologies;
 };
@@ -76,7 +76,7 @@ tracker_ontology_new (void)
 	return ontology;
 }
 
-time_t
+gint64
 tracker_ontology_get_last_modified (TrackerOntology *ontology)
 {
 	TrackerOntologyPrivate *priv;
@@ -103,7 +103,7 @@ tracker_ontology_get_is_new (TrackerOntology *ontology)
 
 void
 tracker_ontology_set_last_modified (TrackerOntology *ontology,
-                                    time_t           value)
+                                    gint64           value)
 {
 	TrackerOntologyPrivate *priv;
 
diff --git a/src/libtracker-data/tracker-ontology.h b/src/libtracker-data/tracker-ontology.h
index b1978b9..f1cad56 100644
--- a/src/libtracker-data/tracker-ontology.h
+++ b/src/libtracker-data/tracker-ontology.h
@@ -51,9 +51,9 @@ struct _TrackerOntologyClass {
 
 GType             tracker_ontology_get_type          (void) G_GNUC_CONST;
 TrackerOntology  *tracker_ontology_new               (void);
-time_t            tracker_ontology_get_last_modified (TrackerOntology *ontology);
+gint64            tracker_ontology_get_last_modified (TrackerOntology *ontology);
 void              tracker_ontology_set_last_modified (TrackerOntology *ontology,
-                                                      time_t           value);
+                                                      gint64           value);
 const gchar *     tracker_ontology_get_uri           (TrackerOntology *ontology);
 gboolean          tracker_ontology_get_is_new        (TrackerOntology *ontology);
 
